src/aui-form-builder/js/aui-form-builder-layout-builder.js

```diff
@@ -210,12 +212,20 @@ A.FormBuilderLayoutBuilder.prototype = {
 
         layout.normalizeColsHeight(layout.get('node').all('.row'));
 
+        this._selectFirstValidMoveTarget();
+
         this._cancelMoveFieldHandles = [
             A.one(A.config.doc).on('click', A.bind(this._onClickOutsideMoveTarget, this)),
             A.one(A.config.doc).on('key', A.bind(this._onEscKeyPressMoveTarget, this), 'down:27')
         ];
     },
 
+    _selectFirstValidMoveTarget: function() {
```

Comment:
Can you fix the alphabetical order here? This should go before _setMoveButtonData